core/test/services/feature-toggle.spec.js

```diff
@@ -19,6 +23,12 @@ describe('Feature Toggle', () => {
       'test'
     ]);
 
```

Comment:
**isDuplicateOrDisabled** and **isValid** seem to be newly added functions. It would be nice to have some unit functions for them as well.